### PR TITLE
Various UX tweaks

### PR DESCRIPTION
**Pull request by j-f1**

### CODE REVIEW

File: client/src/components/feeds/post/content/PostNumber.tsx [hunk 1]
```diff
@@ -8,7 +8,18 @@ interface PostNumberProps {
 function PostNumber(props: PostNumberProps) {
   return (
     <div className="PostNumber">
-      <h3>{props.number ? "#" + props.number : "id: " + props._id}</h3>
+      <h3>
+        {props.number ? (
+          <a
+            href={`/posts/${props.number}`}
```
Comment (DearBlueno) [inline client/src/components/feeds/post/content/PostNumber.tsx]: Since we use React Router, could you change this `<a>` tag to a `<Link>` tag? Also, the href should be going to `/post/#` instead of `/posts/#`.

File: client/src/components/feeds/post/reactions/ReactionButton.css [hunk 2]
```diff
@@ -104,4 +104,7 @@
     width: 1.2rem;
     height: 1.2rem;
   }
+  .PostFooter {
+    min-height: 1.2rem;
+  }
```
Comment (DearBlueno) [inline client/src/components/feeds/post/reactions/ReactionButton.css:109]: It seems like this `min-height` wasn't enough to fix the layout shift on my screen, when I tried with `1.25rem` it seemed to be fixed. Could you try looking into if this value is ideal?

**Review by DearBlueno — CHANGES_REQUESTED** [review]
Thanks for your contributions! There are a couple changes that we need before we can merge this. Other than that, these improvements look great! Congrats on being our first real PR! :)

File: client/src/components/feeds/post/reactions/ReactionButton.css (hunk 2 above)
Comment (j-f1) [reply client/src/components/feeds/post/reactions/ReactionButton.css:109]: I got the height from the height of the buttons (above). I tested in Safari, but it’s possible there’s an issue in whatever browser you’re using. one extra pixel isn’t a huge deal, so I’ll bump it.

File: client/src/components/feeds/post/reactions/ReactionBar.tsx [hunk 3]
```diff
@@ -234,20 +237,25 @@ function ReactionBar(props: ReactionBarProps) {
...
+        );
       }}
     >
       <LoginPopup showPopup={showPopup} closePopup={closePopup} />
       {showReactText && (
         <p
           className="ReactText"
           onClick={
-            props.user
+            props.user || true
```
Comment (DearBlueno) [inline client/src/components/feeds/post/reactions/ReactionBar.tsx]: I assume this was added for debugging and wasn't intended to be included in the commit? If it was intended, why is this necessary?

Comment (j-f1) [reply client/src/components/feeds/post/reactions/ReactionBar.tsx]: Yep, forgot to skip that when committing

**Review by DearBlueno — APPROVED** [review]
Looks good! We'll be merging this and deploying it soon.
